client/App.tsx

```diff
@@ -1,7 +1,25 @@
 import React from "react";
+import { Route, Routes} from "react-router-dom";
+import ZoopsList from './components/ZoopList';
+import LoginPage from "./components/LoginPage";
+import Zoop from "./components/Zoop";
+import RegisterPage from "./components/RegisterPage"
+import MePage from "./components/MePage";
+import HomePage from "./components/HomePage";
 
 const App: React.FC = () => {
-  return <h1>Zoop is cool</h1>;
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<HomePage />}/>
+        <Route path="/zoops" element={<ZoopsList />} />
```

Comment:
One last change - the spec calls for / to display a list of all zoops. Intention here was for ZoopsList to be a child component of HomePage, so no need for separate /zoops route